py-polars/pyproject.toml

```diff
@@ -49,12 +49,13 @@ deltalake = ["deltalake >= 0.10.0"]
 timezone = ["backports.zoneinfo; python_version < '3.9'", "tzdata; platform_system == 'Windows'"]
 matplotlib = ["matplotlib"]
 pydantic = ["pydantic"]
+pyiceberg = ["pyiceberg >= 0.5.0rc3"]
```

Comment:
@stinodego @alexander-beedie All tests are passing. Can we hold off on this PR until 0.5.0 has been released?

Reply:
Don't see why not, as long as you don't mind rebasing if/when necessary, heh 😄
Got a rough ETA on the `0.5.0` release? (Presumably not that far away if it's already at `rc3`).

Reply:
ETA is roughly around early next week